hr_holidays_leave_auto_approve/README.rst

```diff
@@ -0,0 +1,48 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================
+Auto Approve Leaves
+===================
+
+This module allows the user to define a leave type in order to make the system
+automatically approving all the leave requests (and leave allocation requests)
+belonging to that leave type.
+
+Configuration
+=============
+
+Get sure that the administrative user belongs at least to the group
+Holidays/Officer.
+
+If you wish that the system automatically approves all the leave requests
+belonging to a specific leave type, please follow the steps below.
+
+#. Go on the leave type configuration menu
+#. Select the leave type you wish to setup
+#. Mark the flag 'Auto Approve'.
+
```

Comment:
@feketemihai , 
Thank you for reviewing this module. 
In my latest commit I added the missing informations to the readme file.
As for the approve button, I don't think it's a good idea to hide it when the selected status is marked as auto approve, as it would make the request impossible to approve in the following two cases:
- The request was performed, but not approved, before that the module was installed (or after that the module was installed but before that the holiday status was marked as auto approve);
- The administrative user (in contravention with what is suggested in the configuration guide), at the moment of the request creation, doesn't belong to the  Holidays/Officer group.

In both cases, as the functionality is intended to work only during the request creation, the approval will never be automatically triggered and, if we hide the button, also the manual approval won't be possible.
To avoid these issues, and because I don't see any drawback in showing the approval button for auto-approval marked requests, I'd prefer to not add this feature. What's your opinion about that?